src/pages/calendar/components/Section.tsx

```diff
@@ -16,6 +18,33 @@ import { useDarkMode } from 'lib/hooks/useDarkMode';
 import { Schedule } from './Schedule';
 import { SeatInfo } from './Seats';
 
+type BadgeProps = {
+  name: string;
+  color: string;
+};
+
+var names = {
+  'Reserved for BSENG students': 'SENG Only',
+  'Computer Science program.': 'CSC Only',
+  'BEng students': 'ENGR Only',
+  'Faculty of Engineering': 'ENGR/CSC Only',
+  'SCIENCE students': 'SCI only',
+  'BEng and BSEng students': 'BEng/BSeng Only',
+  'BME, BSEN, CENG, ELEC students': 'BME/SENG/CENG/ELEC Only',
+};
```

Comment:
It's unclear from just this to understand what `names` is. If you can't express it in the data structure it might be good to rethink the data structure you're using or see if a comment is needed here.

Reply:
Also, setting it as `var` has a lot of implications. First thing is that you're saying that its going to be modified but in the code there isn't any modifying so it can be a `const`. If you can make a stronger contract about what this value is used for and how you should take it.

In terms of `var` however it's a lot more complicated. You can read about it underneath but in general don't use it unless you are finding a specific need for it.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var
In the majority of cases a `let` or `const` will suffice.

Reply:
In addition, favour code readability over small and perhaps hard to understand variable names too. In terms of performance it gets "transpiled" (which is the process of transforming code into another form of code) anyway so it doesn't really matter.